<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_objectSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>311</width>
       <height>551</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>311</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_applyDefaultSettings">
        <property name="text">
         <string>Apply User Defaults</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_settingsSS" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabWidget::pane {
  border: 1px solid lightgray;
  top:-1px; 
  background: rgb(245, 245, 245);; 
} 

QTabBar::tab {
  background: rgb(230, 230, 230); 
  border: 1px solid lightgray; 
  padding: 5px;
} 

QTabBar::tab:selected { 
  background: rgb(245, 245, 245); 
  margin-bottom: -1px; 
}</string>
        </property>
        <widget class="QGroupBox" name="groupBox_information">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>271</width>
           <height>221</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>Information</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <widget class="QLineEdit" name="lineEdit_objectID">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>141</width>
            <height>20</height>
           </rect>
          </property>
          <property name="toolTip">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Unique object identifier - a prefix will be added with your author ID and object type. Consider the naming guide.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_objectID">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>80</y>
            <width>47</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Object ID</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_author">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>141</width>
            <height>20</height>
           </rect>
          </property>
          <property name="toolTip">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your name, seperate multiple names by comma. It is also good practice to add the name of the original creator(s) if you are fixing/remixing an existing object.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_authorID">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>141</width>
            <height>20</height>
           </rect>
          </property>
          <property name="toolTip">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your identity/name for the object ID. Lowercase recommended.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_author">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>20</y>
            <width>47</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Author</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_authorID">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>50</y>
            <width>47</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Author ID</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_displayName">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>190</y>
            <width>251</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {

	background-color: rgb(230,230, 230);
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="text">
           <string>Display Name</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_suffix">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>160</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Suffix</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_autoNaming">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>110</y>
            <width>141</width>
            <height>17</height>
           </rect>
          </property>
          <property name="toolTip">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable Auto-Naming. Then you can add a global pre- and suffix to your object, every object gets the respective template name added in between. The case 'Fulltile' here is such an example.&lt;/p&gt;&lt;p&gt;If not enabled, then all generated objects will get the &lt;span style=&quot; font-style:italic;&quot;&gt;same&lt;/span&gt; given name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Auto-Naming</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_objectIngameName_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Object ingame name</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_prefix">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>130</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Prefix</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_suffix">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="toolTip">
           <string notr="true">Add suffix here</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_prefix">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="toolTip">
           <string notr="true">The name your object will appear ingame. If you do not tick Auto-Naming every object will be called the same</string>
          </property>
         </widget>
        </widget>
        <widget class="QLineEdit" name="lineEdit_objectType">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>251</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string notr="true">Object type</string>
         </property>
         <property name="text">
          <string>Path Tile Generator</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_remap">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>151</width>
           <height>101</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>Remap</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_remapCheck">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set this check if you want the programm to automatically detect if any of the remaps are used and set the flags accordingly. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Check on Save</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hasPrimaryColour">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Check this if your sprites use the primary remap.</string>
            </property>
            <property name="text">
             <string>Primary Remap</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hasSecondaryColour">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Check this if you want to use the secondary remap as a remap. If not checked, the secondary remap will appear as pink ingame.</string>
            </property>
            <property name="text">
             <string>Secondary Remap</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hasTertiaryColour">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Check this if you want to use the tertiary remap as a remap. If not checked, the tertiary remap will appear as yellow ingame.</string>
            </property>
            <property name="text">
             <string>Tertiary Remap</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_rotation">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_raised">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Set if you want your path to be at the top of clearance (recommended)</string>
         </property>
         <property name="text">
          <string>Raised </string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_rotation">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set if the path should have different sprites for different rotations. Either you give the respective sprites or let the program generate those based on mirroring (thus it is not actual rotation).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Enabled</string>
          </property>
         </item>
        </widget>
        <widget class="QLineEdit" name="lineEdit_outputFolder">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>641</y>
           <width>201</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_importTemplate">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>591</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Import Template</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_clearAllTemplates">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>591</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Clear All</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_OutputFolder">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>621</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Output Folder</string>
         </property>
        </widget>
        <widget class="QListWidget" name="listWidget_templateList">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>411</y>
           <width>251</width>
           <height>171</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_changeOutputFolder">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>640</y>
           <width>31</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_selectAllTemplates">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>591</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_GenerateReturn">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>700</y>
           <width>251</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {

	background-color: rgb(230,230, 230);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_generate">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>670</y>
           <width>141</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Generate Objects</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_remapCheck</tabstop>
  <tabstop>hasPrimaryColour</tabstop>
  <tabstop>hasSecondaryColour</tabstop>
  <tabstop>hasTertiaryColour</tabstop>
  <tabstop>lineEdit_objectType</tabstop>
  <tabstop>lineEdit_author</tabstop>
  <tabstop>lineEdit_authorID</tabstop>
  <tabstop>lineEdit_objectID</tabstop>
  <tabstop>widget_settingsSS</tabstop>
  <tabstop>pushButton_applyDefaultSettings</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
